ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/ip-client-source-interfaces.cfg

```diff
@@ -4,18 +4,18 @@ interface Management1
    vrf MGMT
    ip address 10.73.255.122/24
 !
+ip ftp client source-interface Ethernet10
```

Comment:
I think the templete  is incorrect for this config. Can you please check this config on EOS and we should not fix that in this PR but may be we can create an issue

Reply:
s1-leaf1(config)#sh run | s ip ftp
ip ftp client source-interface Ethernet10